Code/Network/RKHTTPRequestOperation.m

```diff
@@ -112,13 +112,6 @@ - (NSError *)error
 
 #pragma mark - NSURLConnectionDelegate methods
 
-- (void)connection:(NSURLConnection *)connection didReceiveAuthenticationChallenge:(NSURLAuthenticationChallenge *)challenge
```

Comment:
It calls same method on super there. I did not see any super class implementing it, so i thought that it would crash in that case.